nomos/data-availability-layer.md

```diff
@@ -0,0 +1,434 @@
+  
+---
+
+## Abstract
+
+This specification describes the protocol for the data availability for the Nomos network.
+Nomos provides several services for network states to create efficient ecosystems.
+Data availabilty is an important problem that network states need to solved. 
+
+## Background
+
+Nomos is a cluster of blockchains known as zones.
+Zones are layer 2 blockchains that utilize Nomos to maintain sovereignty.
+They are initialized by the Nomos network and can utilize Nomos services, but 
+provide resources on their own.
+They can define their own state as they are sovreign networks.
+Nomos provides tools at the global layer that allow zones to define arbitrary configurations.
+Nomos has two global layers offering services to Zones. 
+The base layer provides data availibility guarantees to zones that choose to utilize it. 
+The second layer is the coordination layer which enables state transition verification through zero-knowledge validity proofs. 
+The base layer allows users with resource-limiting devices, also known as light clients, 
+the ability to obtain all block data and process it locally.
+Light clients should be able to access blockchain data similar to a full node.
+To achieve this, 
+the Nomos data availbilty protocol provides guarantees that transaction data within Nomos zones are vaild.
+
+## Motivation and Goal
+
+Decentralized blockchains require full nodes to verify network transactions by downloading all the data of the network.
+This becomes a problem as the blockchain data grows, full nodes will need more resources to download and 
+store the data while maintaining connection to the network.
+Light nodes on the other hand do not download the entire network data because of it's resource limiting nature.
+This retricts the network from scaling as the network is reliant on full nodes to process transactions.
+and requires light nodes to rely on centralized parties. 
+A blockchain can have light nodes prove the validity of transaction data,
+without the light node to downloading all the blockchain data.
+
+The data availability service on the Nomos base layer is a service that is used by zones for data availability guarantees.
+This allows participants of a zone to access blockchain data in the event that nodes within a zone does not make the data available.
+The service includes data encoding, verification, data availability sampling mechanism, 
+and data retrieval API to solve the data availability problem.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The data availability service of the Nomos base layer consist of different node roles, dispersal clients, 
+data availibity sampling nodes, and data availability provider nodes.
+All network partipants have a choice in which node role they choose to run.
```

Comment:
They do not. All nodes within the base layer *do* sample and verification. Dispersal is triggered by executor in zones, then by all of the base layer nodes participating in the different DA subnetworks (internal protocol yet to be decided).